srcpkgs/ncurses/template

```diff
@@ -93,7 +95,7 @@ do_install() {
 }
 
 ncurses-libs_package() {
-	shlib_provides="libformw.so.5 libmenuw.so.5 libpanelw.so.5 libncursesw.so.5"
+	shlib_provides="libformw.so.5 libmenuw.so.5 libpanelw.so.5 libncursesw.so.5 libtinfo.so.6"
```

Comment:
I believe this is redundant since you also changed `common/shlibs` to resolve this provider for `libtinfo.so.6`.